yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -172,58 +204,155 @@ func backupLogFilesFn() {
 	if !backupLogFiles {
 		return
 	}
-	utils.PrintAndLog("backing up log files")
 	// TODO: in case of failures when cmd is executed again, even if logs were backed up, new log file for end migration will come up
-	err := os.Rename(filepath.Join(exportDir, "logs"), filepath.Join(backupDir, "logs"))
+
+	backupLogDir := filepath.Join(backupDir, "logs")
+	err := os.MkdirAll(backupLogDir, 0755)
+	if err != nil {
+		utils.ErrExit("end migration: creating logs directory for backup: %v", err)
+	}
+
+	utils.PrintAndLog("backing up log files")
```

Comment:
Instead of line 216 to 231: just exec the `mv %s/*.log %s/` command with `logDir, backupLogDir`